src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -605,7 +608,10 @@ private void handlePotentialStuckRepairs(LazyInitializer<Set<String>> busyHosts,
 
           return Pair.of(node, Optional.of(metrics));
         } catch (RuntimeException | ReaperException e) {
-          LOG.debug("failed to query metrics for host {}, trying to get metrics from storage...", node, e);
+          LOG.debug(
+              "failed to query metrics for host {}, trying to get metrics from storage...",
+              node,
+              e);
```

Comment:
trivial: any reason this line changed? i've noticed this a few times with your PRs, maybe your IDE has a unnecessarily short line length configured? speaking up because it makes large patches like this only look bigger…